library/src/webapp/editor/ckextraplugins/common/css/dialog.css

```diff
@@ -0,0 +1,27 @@
+/* search results paging styles */
```

Comment:
Is the use of pattern matching ID selectors so that you don't have to change the HTML?